examples/manifests/mariadb_v1alpha1_mariadb_minimal.yaml

```diff
@@ -3,9 +3,4 @@ kind: MariaDB
 metadata:
   name: mariadb-min
 spec:
-  volumeClaimTemplate:
-    resources:
-      requests:
-        storage: 1Gi
-    accessModes:
-      - ReadWriteOnce
+  ephemeralStorage: true
```

Comment:
Thanks, indeed, it makes sense to update the minimal example with ephemeral storage and empty root password!